modules/reference/pages/rpk/rpk-security/rpk-security-role-describe.adoc

```diff
@@ -0,0 +1,64 @@
+= rpk security role describe
+
+Describe a Redpanda role.
+
+This command describes a `Role`, including the ACLs associated to the role and a list of members that have the role assigned.
```

Comment:
Why is role initcap here?

Reply:
My fault :sweat_smile: I'll correct this in rpk directly along with the other feedback below.